mock_can_data.py

```diff
@@ -0,0 +1,45 @@
+# This script automatically generates and sends CAN messages based off of the DBC file
+# Ensure that you have can and cantools installed, which can be accomplished by running
+# pip install -r requirements.txt
+# Alternatively you could also just run
+# pip install can && pip install cantools
+
+# If you are running this script with virtual CAN ensure that it is set up first
+# You can run the below commands
+# sudo modprobe vcan
+# sudo ip link add dev vcan0 type vcan
+# sudo ip link set up vcan0
+
+import cantools
+import can
+import time
+import random
+
+sleep_time_s = 1
+db = cantools.database.load_file('system_can.dbc')

```

Comment:
add error handling in case it can't find the file with a message like 'generate DBC first!' or similar